android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -281,6 +282,22 @@ public void run() {
         });
     }
 
+    public void refreshTitle_fork(){
+        if( playerControlView == null ){ return; }
```

Comment:
Nit: would be good to keep an eye on formatting. I've seen some irregular spacing.
```java
if (playerControlView == null) { return; }
```

Reply:
Yeah, formatting is kinda all over through this project, so it makes it hard to decide on an approach. Changing all the original fork code would be a pain when dealing with merge conflicts. 

I'll start moving towards that formatting in my own changes though